### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by liliya-dev**
[DEMO LINK](https://liliya-dev.github.io/layout_antisnake/)
[TEST REPORT LINK](https://liliya-dev.github.io/layout_antisnake/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+  <body>
+    <div class="container">
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
```
Comment (V2473) [inline src/index.html:20]: use classes

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+  }
+}
+
+@for $i from 1 through 6 {
+  .container > div:nth-child(#{$i})::after {
+    content: "#{$i}";
+    font-size: 100px;
+    font-family: Arial;
+    color: white;
+    background: rgb((255 - 51*($i - 1)), 0, 0);
```
Comment (V2473) [inline src/styles/main.scss:20]: cool

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
+
+@media screen and (max-width: 899px) and (min-width: 600px) {
+  .container {
+    grid-template-columns: 1fr 1fr;
+  }
+}
```
Comment (V2473) [inline src/styles/main.scss:38]: try make .css file without code repeat

Comment (liliya-dev) [reply src/styles/main.scss:38]: where is repeat? here i make 2 columns

Comment (V2473) [reply src/styles/main.scss:40]: not in scss, but excessive repeats in compiled css

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,40 @@
...
+
+@media screen and (max-width: 599px) {
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
+
+@media screen and (max-width: 899px) and (min-width: 600px) {
+  .container {
+    grid-template-columns: 1fr 1fr;
```
Comment (V2473) [inline src/styles/main.scss:38]: use repeat() instead

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,40 @@
...
+  .container__item:nth-child(#{$i})::after {
+    content: "#{$i}";
+    font-size: 100px;
+    font-family: Arial;
+    color: $white-color;
+    background: rgb((255 - 51*($i - 1)), 0, 0);
+    display: flex;
+    height: 100%;
+    align-items: center;
+    justify-content: center;
```
Comment (V2473) [inline src/styles/main.scss:26]: this should not be in :nth-child(#{$i})::after, 
but in .container__item, 
color in :nth-child(#{$i})

File: src/styles/main.scss [hunk 6]
```diff
@@ -1,3 +1,41 @@
+$white-color: white;
+
 body {
   margin: 0;
 }
+
+.container {
+  display: grid;
+  grid-template-columns: repeat((3), 1fr );
```
Comment (V2473) [inline src/styles/main.scss:9]: parentheses around 3 is excess

File: src/styles/main.scss [hunk 7]
```diff
@@ -1,3 +1,41 @@
...
+
+@media screen and (max-width: 599px) {
+  .container {
+    grid-template-columns: 1fr;
+  }
+}
+
+@media screen and (max-width: 899px) and (min-width: 600px) {
+  .container {
+    grid-template-columns: repeat((2), 1fr );
```
Comment (V2473) [inline src/styles/main.scss:39]: parentheses around 2 is excess

**Pull request closed without merging**